### PR TITLE
[Bots][fix-doc-bug][1492191]

### PR DESCRIPTION
**Pull request by Mahesh2516**
Updated the doc with the Adaptive cards section. Please review

### CODE REVIEW

File: msteams-platform/bots/how-to/conversations/conversation-basics.md [hunk 1]
```diff
@@ -338,6 +338,37 @@ We recommend that you specify the height and width of each image by using XML. I
...
+        "spacing": "extraLarge",
+        "type": "Container",
+        "verticalContentAlignment": "center"
+    }
+    ]
+}
+```
+
+To know more about cards and cards in bots, see [cards documentation](~/task-modules-and-cards/what-are-cards.md).
+When a response contains both text messages and attachments, both responses are sent separately. The text message is sent first, followed by the attachment.
```
Comment (KirtiPereira) [inline msteams-platform/bots/how-to/conversations/conversation-basics.md:370]:
```suggestion
When a response contains text messages and attachments, both responses are sent separately. The attachment is sent after the text message.
```

**Review by KirtiPereira — CHANGES_REQUESTED** [review]
@Mahesh2516 - verify and commit the suggestion.

File: msteams-platform/bots/how-to/conversations/conversation-basics.md [hunk 2]
```diff
@@ -338,6 +338,37 @@ We recommend that you specify the height and width of each image by using XML. I
 * Use - `<img src="http://aka.ms/Fo983c" alt="Duck on a rock" height="150" width="223"></img>`
 * Don't use - `![Duck on a rock](http://aka.ms/Fo983c)`
 
+## Adaptive cards
+
```
Comment (Trinetra-MSFT) [inline msteams-platform/bots/how-to/conversations/conversation-basics.md:342]: Looks Good

**Review by Trinetra-MSFT — DISMISSED** [review]
You can Merge it

**Review by KirtiPereira — DISMISSED** [review]
Approved

**Review by surbhigupta12 — DISMISSED** [review]
Approving the PR

**Review by KirtiPereira — DISMISSED** [review]
Approved after resolved conflicts.

**Review by KirtiPereira — APPROVED** [review]
Approve

**Review by surbhigupta12 — APPROVED** [review]
Approving the PR

**Pull request merged**